levels/grandmas-awakening.json

```diff
@@ -923,6 +927,125 @@
-
+      {
+         "object-name": "fire hydrant",
+         "object-type": "DYNAMIC_OBJECT",
+         "pos": {
+            "x": 0.0,
+            "y": 1.0,
+            "z": 0.0
+         },
+         "dir": {
+            "x": 0.0,
+            "y": 1.0,
+            "z": 0.0
+         },
+         "vel": 0.0,
+         "scale": {
+            "x": 1.0,
+            "y": 1.0,
+            "z": 1.0
+         },
+         "physics-component": {
+            "name": "FireHydrantPhysicsComponent"
+         },
+         "render-component": {
+            "name": "WallRenderComponent",
+            "shape": "FireHydrant",
+            "shader": "Red",
+            "material": "Obsidian"
+         },
+         "deliverable": false
+      },
+      {
+         "object-name": "fire hydrant",
+         "object-type": "DYNAMIC_OBJECT",
+         "pos": {
+            "x": 0.0,
+            "y": 1.0,
+            "z": -2.0
+         },
+         "dir": {
+            "x": 0.0,
+            "y": 1.0,
+            "z": 0.0
+         },
+         "vel": 0.0,
+         "scale": {
+            "x": 1.0,
+            "y": 1.0,
+            "z": 1.0
+         },
+         "physics-component": {
+            "name": "FireHydrantPhysicsComponent"
+         },
+         "render-component": {
+            "name": "WallRenderComponent",
+            "shape": "FireHydrant",
+            "shader": "Red",
+            "material": "Obsidian"
+         },
+         "deliverable": false
+      },
+      {
+         "object-name": "fire hydrant",
+         "object-type": "DYNAMIC_OBJECT",
+         "pos": {
+            "x": 0.0,
+            "y": 1.0,
+            "z": -4.0
+         },
+         "dir": {
+            "x": 0.0,
+            "y": 1.0,
+            "z": 0.0
+         },
+         "vel": 0.0,
+         "scale": {
+            "x": 1.0,
+            "y": 1.0,
+            "z": 1.0
+         },
+         "physics-component": {
+            "name": "FireHydrantPhysicsComponent"
+         },
+         "render-component": {
+            "name": "WallRenderComponent",
+            "shape": "FireHydrant",
+            "shader": "Red",
+            "material": "Obsidian"
+         },
+         "deliverable": false
+      },
+      {
+         "object-name": "fire hydrant",
+         "object-type": "DYNAMIC_OBJECT",
+         "pos": {
+            "x": 0.0,
+            "y": 1.0,
+            "z": 2.0
+         },
+         "dir": {
+            "x": 0.0,
+            "y": 1.0,
+            "z": 0.0
+         },
+         "vel": 0.0,
+         "scale": {
+            "x": 1.0,
+            "y": 1.0,
+            "z": 1.0
+         },
+         "physics-component": {
+            "name": "FireHydrantPhysicsComponent"
+         },
+         "render-component": {
+            "name": "WallRenderComponent",
+            "shape": "FireHydrant",
+            "shader": "Red",
+            "material": "Obsidian"
+         },
+         "deliverable": false
+      }
```

Comment:
Do we have a red material? I can always look one up. I think I have one actually from my old 471 stuff. Will look into it.